action.yml

```diff
@@ -119,6 +119,7 @@ runs:
             fi
             echo "Nix was already installed, using existing install"
             echo "/nix/var/nix/profiles/default/bin" >> $GITHUB_PATH
+            echo "$HOME/.nix-profile/bin" >> $GITHUB_PATH
```

Comment:
If nix isn't installed, we do this in the installer: https://github.com/DeterminateSystems/nix-installer/blob/79a32a7c668af6936f272960dbab80b05019bd16/src/action/common/configure_shell_profile.rs#L151-L170

If the user installed Nix but hasn't run a Nix command yet, that path might not exist. Then if they ran a Nix command (eg `nix profile install nixpkgs#hello`) I believe that file would be created and it wouldn't work since it wasn't on the path?